File: ScalaPerformanceTest/src/akshay/scalaperformance/RandomString.scala
package akshay.scalaperformance

import java.util.Random
import java.lang.Long

class RandomString(i: Int, j: Int){
    val r = new Random
    val l = Math abs r.nextLong
    val s = java.lang.Long.toString(l,36)
    var num = i + j
}

File: JavaPerformanceTest/src/akshay/javaperformance/RandomString.java
package akshay.javaperformance;

import java.util.Random;

public class RandomString {
	
	private String s;
	private int num;

	public RandomString(int i, int j){
    Random r = new Random();
    long l = Math.abs(r.nextLong());
    this.s = java.lang.Long.toString(l,36);
    this.num = i + j;
	}
	
	public int getNum() {
		return num;
	}

	public String getS() {
		return s;
	}
}